packages/block-library/src/blocks/hero/deprecated.js

```diff
@@ -0,0 +1,109 @@
+import { addFilter } from "@wordpress/hooks";
+import { createBlock } from '@wordpress/blocks';
+
+import { omit } from 'lodash';
+
+import save from "./save";
+
+const heroAddDeprecated = (settings, name) => {
+
+  if ( name !== 'novablocks/hero' ) {
+    return settings;
+  }
+
+  const attributes = settings.attributes;
+
+  const deprecated = [
```

Comment:
Could this be split into two separate functions and the one deprecating `contentPadding` be applied on both hero and slideshow without duplicating it?